staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -178,3 +184,168 @@ func ValidateMetrics(metrics Metrics, metricName string, expectedLabels ...strin
 	}
 	return nil
 }
+
+// Histogram wraps prometheus histogram DTO (data transfer object)
+type Histogram struct {
+	histogram *dto.Histogram
+}
```

Comment:
Change to

```
type Histogram struct {
	*dto.Histogram
}
```

embedding will allow the other PR to create your Histogram wrapper from a dto.Histogram

Reply:
Done